lib/node/failure_detector.ml

```diff
@@ -82,27 +82,28 @@ let update_peer_status node peer status =
       (Printf.sprintf "Failed to find peer with address %s:%d in node peer list"
          peer.address.address peer.address.port)
 
-let create_message node message (recipient : Peer.t) =
+let create_message node message (recipients : Peer.t list) =
```

Comment:
Hum, I am not sure about this change.
In SWIM paper, a message is sent to only one recipient.
Only the messages about node state (for example "a peer is suspicious") are disseminated. The others, are kinda request/response via direct messaging.